searchlib/src/vespa/searchlib/queryeval/blueprint.h

```diff
@@ -164,6 +164,8 @@ class Blueprint
         virtual bool check(const Blueprint & bp) const = 0;
     };
 
+    enum class FilterBound { UPPER, LOWER };
```

Comment:
I think the 'bound' part should be part of the enum values and not the enum type, since it smells like past tense 'bind' on its own. Maybe FilterType or FilterConstraint, with UPPER_BOUND and LOWER_BOUND

Reply:
Also add a comment describing the difference between the two.